pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -223,13 +223,20 @@
 @click.option('-v', '--verbose/--no-verbose',
               default=None,
               help='Display extra information about the processing.')
+@click.option('--pdb', is_flag=True,
+              # defaulted in code
+              envvar=PywbemServer.pdb_envvar,
+              help='Break in the debugger before executing the command. '
```

Comment:
Change the help to "the debugger will pause execution just before executing the command"

Reply:
DONE, using this help text: "Pause execution in the built-in pdb debugger just before executing the command within pywbemcli."